src/Support/helpers.php

```diff
@@ -210,13 +227,31 @@ function temp_path($path = '')
     }
 }
 
+if (!function_exists('e')) {
```

Comment:
Did Laravel get rid of this helper and now we have to implement it ourselves?

Reply:
No, they didn't remove it, but they set the `$doubleEncode` parameter to `true` by default in 5.6 which stuffed up a lot of October functionality, so I opted to define the helper ourselves with the original `false` default.

Reply:
https://github.com/octobercms/library/commit/adda4366544e9e690b1a2b6d0cf3011df8a9ca3a